GitUI/UserControls/BlameControl.cs

```diff
@@ -414,7 +414,18 @@ private int GetBlameLine()
 
         private void contextMenu_Opened(object sender, EventArgs e)
         {
+            blameRevisionToolStripMenuItem.Enabled = false;
```

Comment:
I was confused enough to comment and would like to have a comment like
//Disable the menu entries by default
Disabling could be done in the early exit too

Reply:
I put it back in the early exit as it is equivalent and was what I did first...